ml-agents/mlagents/trainers/tf/components/bc/module.py

```diff
@@ -106,15 +106,19 @@ def _update_batch(
             self.policy.batch_size_ph: n_sequences,
             self.policy.sequence_length_ph: self.policy.sequence_length,
         }
-        feed_dict[self.model.action_in_expert] = mini_batch_demo["actions"]
         if self.policy.behavior_spec.action_spec.is_discrete():
+            feed_dict[self.model.action_in_expert] = mini_batch_demo["discrete_action"]
             feed_dict[self.policy.action_masks] = np.ones(
                 (
                     self.n_sequences * self.policy.sequence_length,
                     sum(self.policy.behavior_spec.action_spec.discrete_branches),
                 ),
                 dtype=np.float32,
             )
+        else:
+            feed_dict[self.model.action_in_expert] = mini_batch_demo[
+                "continuous_action"
+            ]
```

Comment:
else raise("Not implemented yet for Hybrid Actions")

Reply:
Can we hold off on this since its not going into master until the ActionModel is merged?